_data/sites.yml

```diff
@@ -1993,6 +1993,11 @@
   size: 91.1
   last_checked: 2022-03-21
 
+- domain: saucecode.bar
+  url: https://saucecode.bar/
+  size: 5.6
```

Comment:
Nitpick: this should be 5.61, but I'll let it slide. ;)

Reply:
Oops, Thanks.

Reply:
I updated this size as well.